GitCommands/RevisionReader.cs

```diff
@@ -339,23 +337,24 @@ long ParseUnixDateTime(in ReadOnlySpan<byte> array)
 
             #region Encoded string values (names, emails, subject, body)
 
-            // Finally, decode the names, email, subject and body strings using the required text encoding
-            ReadOnlySpan<char> s = _logOutputEncoding.GetString(array[offset..]).AsSpan();
-            StringLineReader reader = new(in s);
+            // The remaining must be decoded (for above utf8/ascii must work)
+            Span<char> decoded = stackalloc char[_logOutputEncoding.GetMaxByteCount(array.Slice(offset).Length)];
+            _logOutputEncoding.GetChars(array.Slice(offset), decoded);
```

Comment:
This is the primary improvement in this PR.
The execution time is not much different for this part, 50 -80 ms for me for loading GE. (Finding size of what is needed to allocate on the stack is comparable with the allocation of the string.)
However there are typically 5 instead of typically 6 garbage collections when loading so the timing is fluctuating less.